Docker build failed

The last nupic image available from [docker hub](https://hub.docker.com/r/numenta/nupic/builds/) is 3 months old and when I try to build a new image using:
 ```
docker build .
````

I get the following error:

```
...

Searching for pycapnp==0.5.5
Reading https://pypi.python.org/simple/pycapnp/
Best match: pycapnp 0.5.5
Downloading https://pypi.python.org/packages/source/p/pycapnp/pycapnp-0.5.5.tar.gz#md5=71570126bbec8b38f992b1e8feebaeab
Processing pycapnp-0.5.5.tar.gz
Writing /tmp/easy_install-BsUy7X/pycapnp-0.5.5/setup.cfg
Running pycapnp-0.5.5/setup.py -q bdist_egg --dist-dir /tmp/easy_install-BsUy7X/pycapnp-0.5.5/egg-dist-tmp-v3zfsP
/tmp/easy_install-BsUy7X/pycapnp-0.5.5/temp/tmpnWYf1T/tmp/easy_install-BsUy7X/pycapnp-0.5.5/temp/timer_create3atbhu.o: In function `main':
timer_create3atbhu.c:(.text+0x15): undefined reference to `timer_create'
collect2: error: ld returned 1 exit status
/tmp/easy_install-BsUy7X/pycapnp-0.5.5/temp/tmpnWYf1T/vers.cpp:4:26: fatal error: capnp/common.h: No such file or directory
 #include "capnp/common.h"
                          ^
compilation terminated.
*WARNING* no libcapnp detected. Will download and build it from source now. If you have C++ Cap'n Proto installed, it may be out of date or is not being detected. Downloading and building libcapnp may take a while.
fetching https://capnproto.org/capnproto-c++-0.5.1.2.tar.gz into /tmp/easy_install-BsUy7X/pycapnp-0.5.5/bundled
error: Setup script exited with error: [Errno 2] No such file or directory
The command '/bin/sh -c python setup.py install' returned a non-zero code: 1
```
CC: @oxtopus

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,7 +4,7 @@ MAINTAINER Allan Costa <allaninocencio@yahoo.com.br>
 
 # Install dependencies
 RUN apt-get update
-RUN apt-get install -y wget git-core gcc g++ python2.7 python 2.7-dev
+RUN apt-get install -y wget git-core build-essential python2.7 python 2.7-dev
 RUN wget https://raw.github.com/pypa/pip/master/contrib/get-pip.py -O - | python
 RUN pip install --upgrade setuptools
 RUN pip install wheel
@@ -12,6 +12,14 @@ RUN pip install wheel
 # Use gcc to match nupic.core binary
 ENV CC gcc
 ENV CXX g++
+
+# Install capnproto from source
+RUN wget https://capnproto.org/capnproto-c++-0.5.2.tar.gz && \
+    tar zxf capnproto-c++-0.5.2.tar.gz && \
+    cd capnproto-c++-0.5.2 && \
+    ./configure && \
+    make check && \
+    make install
 
 # Set enviroment variables needed by NuPIC
 ENV NUPIC /usr/local/src/nupic

